docs/using.rst

```diff
@@ -119,7 +119,7 @@ For more information about the layout
 of the ``/etc/letsencrypt`` directory, see :ref:`where-certs`.
 
 .. _Docker: https://docker.com
-.. _`install Docker`: https://docs.docker.com/userguide/
+.. _`install Docker`: https://docs.docker.com/engine/installation/
```

Comment:
`https://docs.docker.com/engine/userguide/intro/` is different from `https://docs.docker.com/engine/installation/` though? Looks like `https://docs.docker.com/engine/userguide/intro/` returns 200, do we want to be directing people to the high-level install docker page?